packages/adapter-cloudflare-workers/index.js

```diff
@@ -8,15 +8,11 @@ import { fileURLToPath } from 'url';
  * @typedef {import('esbuild').BuildOptions} BuildOptions
  */
 
-/**
- * @param {{
- *   esbuild?: (defaultOptions: BuildOptions) => Promise<BuildOptions> | BuildOptions;
- * }} [options]
- **/
+/** @type {import('.')} */
 export default function (options) {
-	/** @type {import('@sveltejs/kit').Adapter} */
```

Comment:
I'm not sure about removing this. When I open up `adapter-static` in VS Code in `master` it complains: 
>Binding element 'utils' implicitly has an 'any' type